### PR TITLE
Find a Rep 2.0 Design/styling updates

### PR DESCRIPTION
**Pull request by cosu419**
## Description

This PR contains UI styling changes to implement the latest Find a Rep desktop designs. Mobile view is still in progress. 

image

image

image
 

## Summary

- Result card styled to approximate design (va-icons are under construction)
- Fixed issue with frontend passing type of `VSO` (rather than `veteran_service_officer`)
- Some refactoring - ReportModal is now its own component, SearchPage now has render method for search controls.
- In ReportModal, set UI behaviors for validation errors when submitting a report (blank fields)
- Added disclaimer banner to results section
- Updated copy in additional-info components 
- Added "Accredited claims agent" rep type + additional-info component

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74244
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74215
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74338
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74333
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74330
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74215
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/75075

## Testing done

- Several broken tests were repaired, but holding off on writing new tests until mobile view is implemented + all controls are in place (search radius dropdown for example)

## Screenshots
image

image

image

### CODE REVIEW

**Review by holdenhinkle — DISMISSED** [review]
LGTM!

File: src/applications/representative-search/config.js [hunk 1]
```diff
@@ -67,7 +68,7 @@ export const resolveParamsWithUrl = ({
   page,
   perPage = 10,
   sort,
-  type = 'VSO',
+  type = 'veteran_service_officer',
```
Comment (holdenhinkle) [inline src/applications/representative-search/config.js:71]: Maybe grab the type string from the `RepresentativeType` constant?

**Review by holdenhinkle — COMMENTED** [review]
Just a thought as I quickly reviewed the code.

What do you think of making better use of the `RepresentativeType` constant to grab the values for whatever `VETERAN_SERVICE_OFFICER` is set to? Same for the other types. I see `veteran_service_officer` lots of places so grabbing that value from the constant would allow you to more easily update it (like I think you had to recently). Just a thought.

**Review by holdenhinkle — APPROVED** [review]
We discussed my comments in Slack.

LGTM for now.

File: src/applications/representative-search/config.js (hunk 1 above)
Comment (cosu419) [reply src/applications/representative-search/config.js:71]: Will include in next PR.

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**